tests/test_examples.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# Copyright 2020, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+""" Unit tests for examples scripts.
+
+"""
+import glob
+import os
+import shutil
+import tempfile
+import unittest
+from pathlib import Path
+
+
+class TestRepoExamples(unittest.TestCase):
+    """Unit test class for repo example scripts.
+
+    Provides a '_run_example_script' method to run (exec) a script located in
+    the 'repo example' directory.
+
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Locate and cache repo example dir."""
+        base = Path(__file__).resolve().parents[1]
+        cls.repo_examples_dir = base / "examples" / "repo_example"
```

Comment:
Should we use `os.path.join()` or do we not want to run this on Windows?

Reply:
This actually works on Windows too, because `pathlib` overloads the `/` operator to make it act like `join`.

https://docs.python.org/3/library/pathlib.html#operators

Reply:
Oh, wow. TIL. I cannot decide whether I like this magic or not 😆